mantle/platform/metal.go

```diff
@@ -419,7 +419,7 @@ func (t *installerRun) completePxeSetup(kargs []string) error {
 		// s390 doesn't seem to have a pre-created pxe image although have to check on this
 		if t.pxe.pxeimagepath == "" {
 			kernelpath := filepath.Join(t.builddir, t.kern.kernel)
-			initrdpath := filepath.Join(t.builddir, t.kern.initramfs)
+			initrdpath := filepath.Join(t.tftpdir, t.kern.initramfs)
```

Comment:
It looks like we should have been doing this the whole time, right?

We should also change kernelpath to use tftpdir too IMO.